sale_order_tag/models/sale_order_tag.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2019 Pavlov Media
+# License Proprietary. Do not copy, share nor distribute.
+
+from odoo import fields, models
+
+
+class SaleOrderTags(models.Model):
+    _name = "sale.order.tag"
+    _order = "name"
+    _description = "Sale Order Tag"
+
+    name = fields.Char(required=True)
+    description = fields.Text(string="Description")
+    color = fields.Integer(string="Color Index")
+    team_ids = fields.Many2many('crm.team', string="Sales Teams")
+    active = fields.Boolean(string="Active", default=True)
+    sequence = fields.Integer(
```

Comment:
Could you add a company fields, optional to make this module working in a multi company context ?
(With the according ir.rule file)